src/maker/config.rs

```diff
@@ -61,22 +61,27 @@ impl MakerConfig {
             if path.exists() {
                 parse_toml(path)?
             } else {
+                create_default_maker_dirs(&path);
                 log::warn!(
-                    "Maker config file not found at path : {}, using default config",
+                    "Maker config file not found, creating a default file at path : {}, ",
                     path.display()
                 );
-                HashMap::new()
+                parse_toml(&path)?
             }
         } else {
             let default_path = PathBuf::from("maker.toml");
             if default_path.exists() {
                 parse_toml(&default_path)?
```

Comment:
This code should is not needed anymore. There is no need for the config files at repo dir, As they will always be created at runtime in default/custom datadir if doesn't exist. 

We should document this in our readme. As well as in the function doc.

Reply:
ok